misc/ide/jetbrains/build.gradle

```diff
@@ -0,0 +1,108 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.3.2'
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+apply plugin: 'com.android.library'
+
+dependencies {
+    implementation "com.android.support:support-core-utils:28.0.0"
+}
+
+def pathToRootDir = "../../../"
+// Note: Only keep the abis you support to speed up the gradle 'assemble' task.
+def supportedAbis = ["armv7", "arm64v8", "x86"]
```

Comment:
Why not `x86_64`?

Reply:
I originally went by the documentation [here](https://docs.godotengine.org/en/latest/development/compiling/compiling_for_android.html#adding-support-for-x86-devices), hence only these three since those are the ones mentioned there.

The `godot/platform/android/SCub` build file shows support for `armv6` and `x86_64` as well, so I'll add those as well.

Reply:
Btw, the `armv6` build support seems to be broken. I get the following error messages when attempting to build it:
```
/var/folders/hf/nggb0mq11hv4t5s0vzfvs_8m00bx2l/T/thread_jandroid-2bb15c.s: Assembler messages:
/var/folders/hf/nggb0mq11hv4t5s0vzfvs_8m00bx2l/T/thread_jandroid-2bb15c.s:119: Error: selected processor does not support `ldrexd r2,r3,[r0]' in ARM mode
/var/folders/hf/nggb0mq11hv4t5s0vzfvs_8m00bx2l/T/thread_jandroid-2bb15c.s:122: Error: selected processor does not support `strexd r1,r6,r7,[r0]' in ARM mode
/var/folders/hf/nggb0mq11hv4t5s0vzfvs_8m00bx2l/T/thread_jandroid-2bb15c.s:322: Error: selected processor does not support `ldrexd r2,r3,[r0]' in ARM mode
/var/folders/hf/nggb0mq11hv4t5s0vzfvs_8m00bx2l/T/thread_jandroid-2bb15c.s:325: Error: selected processor does not support `strexd r1,r4,r5,[r0]' in ARM mode
```

Seems as far as [Aug 2017](https://github.com/godotengine/godot/issues/10711) it was recommended to use `armv7` instead. Should we consider `armv6` to be deprecated?